src/openrct2/peep/Peep.cpp

```diff
@@ -339,6 +344,16 @@ Peep* rct_sprite::AsPeep()
     return result;
 }
 
+Vehicle* rct_sprite::AsVehicle()
+{
+    Vehicle* result = nullptr;
+    if (IsVehicle())
+    {
+        return reinterpret_cast<Vehicle*>(this);
+    }
+    return result;
+}
```

Comment:
Same as the previous comment.

Reply:
I wasn't using it in the end, so have removed it.